plugins/services/src/js/service-configuration/ServiceNetworkingConfigSection.js

```diff
@@ -169,6 +171,33 @@ class ServiceNetworkingConfigSection extends ServiceConfigBaseSectionDisplay {
               });
             }
 
+            const containsServicePorts = portDefinitions.filter(function(
```

Comment:
Let's use `some` since we don't really need `portDefinitions` that contain a `servicePort`